stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -52,6 +52,7 @@
 public class Coordinator {
 
     public final static File STABILIZER_HOME = getStablizerHome();
+    public final static String RESOURCES_HOME = STABILIZER_HOME.getAbsolutePath() + "/resources";
```

Comment:
Why do we need this? We only upload the 'upload' directory if there is something in the working directory. Why do we need to have a resources directory in stabilizer?